ENG-10447: Changes method to clone demo notebooks (#392)
opendatahub-io/opendatahub-documentation: modules/running-distributed-data-science-workloads-from-notebooks.adoc

. Download the demo notebooks provided by the CodeFlare SDK.
The demo notebooks provide guidelines for how to use the CodeFlare stack in your own notebooks.
+
To access the demo notebooks, run the `codeflare_sdk.copy_demo_nbs()` function in a Jupyter notebook.

> The function identifies the demo notebooks that are compatible with the currently installed version of the CodeFlare SDK, and clones them into the target location. 

Reviewer: ```suggestion
The function copies the demo notebooks that are packaged with the currently installed version of the CodeFlare SDK, and clones them into the target location. 
```

Author: Done, thanks!

+
The `codeflare_sdk.copy_demo_nbs()` function accepts two arguments:

> * `dir(str)` specifies the target location for the cloned demo notebooks. 

Reviewer 2: ```suggestion
* `dir: str` specifies the target location for the cloned demo notebooks. 
```

that's the usual Python convention for specifying a type

If the leading character is a forward slash (/), the path is absolute. 
Otherwise, the path is relative to the current directory.
The default value is `"./demo-notebooks"`.

* `overwrite(bool)` allows the function to overwrite the contents of an existing directory.

> When this argument is set to `True`, the function overwrites any customizations that you made in the original demo files.

Reviewer 3: I'm just wondering if you we can cut out "that you made" here.

The default value is `False`.

+
Examples:

+

> ----
> codeflare_sdk.copy_demo_nbs("my-dir", True)
> codeflare_sdk.copy_demo_nbs("my-dir")
> codeflare_sdk.copy_demo_nbs(dir="my-dir")
> codeflare_sdk.copy_demo_nbs(overwrite=True)
> codeflare_sdk.copy_demo_nbs(dir="my-dir", overwrite=True)
> codeflare_sdk.copy_demo_nbs(overwrite=True, dir="my-dir")
> codeflare_sdk.copy_demo_nbs("/absolute/path/to/target-dir")
> ----

Reviewer 2: do we expect the user to be familiar with Python? lot's of the examples are just variations on Python syntax (such as that keyword arguments can be given in any order). I'd cut down on the number of examples here

```suggestion
----
codeflare_sdk.copy_demo_nbs("my-dir")
codeflare_sdk.copy_demo_nbs("/absolute/path/to/target-dir")
codeflare_sdk.copy_demo_nbs(dir="my-dir", overwrite=True)
----
```
